### ISSUE TITLE
How do I open/close tabs?

### ISSUE THREAD

**Issue by angularsen** [lunixbochs/ActualVim#126]
`ctrl+n` no longer opens a new tab and `ctrl+w` no longer closes the current tab.
How can I either pass-through these shortcuts to sublime to get original behavior, or achieve the same with vim key bindings?

Update: `ctrl+f4` closes tab.

**Comment by DarrienG** [comment]
:q and :e and :vspl also don't work for this either.

**Comment by lunixbochs** [comment]
I do detect when a new buffer unexpectedly appears on the vim side so it might be easy to map that to opening a tab. My answer personally is that ActualVim is very nice on mac where the hotkeys don't overlap.

Another option is for me to expose Sublime commands generically to vim and recommend mapping ^N to new tab while in command mode (which won't conflict with the insert mode autocomplete). I think this is the best option.

**Comment by angularsen** [comment]
Most other editors I use vim extensions with (Visual Studio, VS Code) that are *not* using neovim offer a way to disable certain vim key bindings - in order to pass the key combination down to the editor instead and let it do it's default thing. I think this is the better solution, but I'm not sure how neovim supports this scenario.

**Comment by lunixbochs** [comment]
Okay, here's how I'd do that:

ActualVim user settings allow easily changing view settings based on the Vim mode.
Sublime keymaps can be disabled based on view settings.
So: I can add a context to each keymap (i.e. `ctrl-c`) like `not av_host_ctrl_c`, then you can toggle these hotkeys for every Vim mode, or for specific Vim modes by putting `av_host_ctrl_c=true` in your config.

**Comment by angularsen** [comment]
That sounds like a good solution to me; to be able to skip neovim for certain key combinations (`ctrl+c`, `ctrl+shift+n` etc) for one or more vim modes (`normal`, `insert`, `visual`).

**Comment by lunixbochs** [comment]
Fixed in 0.9.12, which adds this setting and on non-osx hosts disables ^N and ^W passthrough.

Since you've already run ActualVim it probably won't inject this setting for you, so you'll need to add a setting like this in your ActualVim settings:

At the path settings.vim.modes
```
"normal": {
        "av:ctrl+n": true,
        "av:ctrl+w": true,
}
```
